### PR TITLE
Improve account identity

### PR DESCRIPTION
**Pull request by Nikita-Polyakov**
### **Description**
- **What does this PR do?**
  - Add Identity component with on chain data
  - Show identities on WalletSend component
  
- **Why is this change needed?**
  - Feature request to have a proof of address

### **Type of change**
- [ ] Bug fix
- [x] New feature
- [x] Refactor (cleaning up code without changing functionality)
- [ ] Documentation update
- [ ] Tests
- [ ] Other (describe below):

---

### **Checklist**
- [x] Code adheres to coding guidelines and standards.
- [x] Linting and formatting checks have passed, no local issues.
- [x] Existing tests pass successfully, no local errors.
- [x] Pull request is linked to a relevant issue or task.
- [ ] Tests are written for new features or fixes.
- [ ] Documentation has been updated (if applicable).
  
---

### **Related Issue/Task**
- Issue/Task link: https://soramitsu.atlassian.net/browse/PW-1803

---

### **Testing**
- **How has this been tested?**
  - _Describe manual testing steps or link to relevant automated tests._
  
- **Edge cases covered:**
  - _List any specific cases handled by this PR._

---

### **Screenshots/Video/Link to dedicated environment (if applicable)**
- 
identity

---

### **Additional Notes**
- _Add any information the reviewer should consider, especially cross-project impacts, UX changes, or potential risks._

---

### **Reviewer Tasks**
- [ ] Review logic for correctness and clarity.
- [ ] Verify that the code is following best practices.
- [ ] Test or review related areas if this PR affects other projects.

### CODE REVIEW

File: src/components/Account/Identity.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.account-identity {
+  display: flex;
+  align-items: center;
+  gap: $basic-spacing-mini;
+
+  &-status {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 16px;
```
Comment (RustemYuzlibaev) [inline src/components/Account/Identity.vue:76]: should be variables, most likely $basic-spacing, but px values are not the same across wallet and polkaswap for some reason

**Review by RustemYuzlibaev — APPROVED** [review]

**Review by RustemYuzlibaev — APPROVED** [review]
